### PR TITLE
fix: remove floating buttons from main element in SfScrollable 

### PR DESCRIPTION
**Pull request by AdamPawlinski**
# Related issue

Closes[ #1073](https://vsf.atlassian.net/browse/SFUI2-1073)

# Scope of work

Remove floating buttons from the inside of the main element in SfScrollable. 

# Screenshots of visual changes

# Checklist

- [x] Self code-reviewed
- [x] Changes documented
- [x] Semantic HTML
- [x] SSR-friendly
- [x] Caching friendly
- [x] a11y for WCAG 2.0 AA
- [x] examples created
- [x] blocks created
- [x] cypress tests created

### CODE REVIEW

File: packages/sfui/frameworks/react/components/SfScrollable/SfScrollable.tsx [hunk 1]
```diff
@@ -110,14 +109,14 @@ const SfScrollable = polymorphicForwardRef<typeof defaultScrollableTag, SfScroll
...
-          />
-        )}
+        <PreviousButton
+          classNameButton={classNames('!rounded-full bg-white', {
+            'mr-4': isBlock && isHorizontal,
+            'mb-4 rotate-90': isBlock && !isHorizontal,
+            'absolute left-4 z-10': isFloating && isHorizontal,
+            'absolute top-4 rotate-90 z-10': isFloating && !isHorizontal,
+          })}
+        />
```
Comment (FRSgit) [inline packages/sfui/frameworks/react/components/SfScrollable/SfScrollable.tsx:119]:
```suggestion
        {(isBlock || isFloating) && <PreviousButton
          classNameButton={classNames('!rounded-full bg-white', {
            'mr-4': isBlock && isHorizontal,
            'mb-4 rotate-90': isBlock && !isHorizontal,
            'absolute left-4 z-10': isFloating && isHorizontal,
            'absolute top-4 rotate-90 z-10': isFloating && !isHorizontal,
          })}
        />
        }
```

File: packages/sfui/frameworks/react/components/SfScrollable/SfScrollable.tsx [hunk 2]
```diff
@@ -129,32 +128,16 @@ const SfScrollable = polymorphicForwardRef<typeof defaultScrollableTag, SfScroll
...
-          />
-        )}
+        <NextButton
+          classNameButton={classNames('!rounded-full bg-white', {
+            'ml-4': isHorizontal,
+            'mt-4 rotate-90': !isHorizontal,
+            'absolute right-4 z-10': isFloating && isHorizontal,
+            'absolute bottom-4 rotate-90 z-10': isFloating && !isHorizontal,
+          })}
+        />
```
Comment (FRSgit) [inline packages/sfui/frameworks/react/components/SfScrollable/SfScrollable.tsx:140]:
```suggestion
        {(isBlock || isFloating) && <NextButton
          classNameButton={classNames('!rounded-full bg-white', {
            'ml-4': isHorizontal,
            'mt-4 rotate-90': !isHorizontal,
            'absolute right-4 z-10': isFloating && isHorizontal,
            'absolute bottom-4 rotate-90 z-10': isFloating && !isHorizontal,
          })}
        />}
```

File: packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue [hunk 3]
```diff
@@ -86,13 +86,15 @@ const isHorizontal = computed(() => props.direction === SfScrollableDirection.ho
   <div :class="['items-center', 'relative', isHorizontal ? 'flex' : 'flex-col h-full inline-flex', wrapperClass]">
     <div v-if="$slots.previousButton" v-bind="getPrevButtonProps"><slot name="previousButton" /></div>
     <SfButton
-      v-else-if="buttonsPlacement === SfScrollableButtonsPlacement.block"
+      v-else-if="buttonsPlacement"
```
Comment (FRSgit) [inline packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue:89]:
```suggestion
      v-else-if="buttonsPlacement !== SfScrollableButtonsPlacement.none"
```

File: packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue [hunk 4]
```diff
@@ -86,13 +86,15 @@ const isHorizontal = computed(() => props.direction === SfScrollableDirection.ho
...
       variant="secondary"
       size="lg"
       square
       :class="[
         '!rounded-full bg-white hidden md:block',
-        isHorizontal ? 'mr-4' : 'mb-4 rotate-90',
+        isHorizontal && SfScrollableButtonsPlacement.block ? 'mr-4' : 'mb-4 rotate-90',
```
Comment (FRSgit) [inline packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue:95]:
```suggestion
        isHorizontal && buttonsPlacement === SfScrollableButtonsPlacement.block ? 'mr-4' : 'mb-4 rotate-90',
```

File: packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue [hunk 5]
```diff
@@ -86,13 +86,15 @@ const isHorizontal = computed(() => props.direction === SfScrollableDirection.ho
...
+        isHorizontal && SfScrollableButtonsPlacement.floating ? 'left-4' : 'top-4 rotate-90',
```
Comment (FRSgit) [inline packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue:96]:
```suggestion
       buttonsPlacement 
 === SfScrollableButtonsPlacement.floating && (isHorizontal ? 'left-4' : 'top-4 rotate-90'),
```

File: packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue [hunk 6]
```diff
@@ -114,48 +116,19 @@ const isHorizontal = computed(() => props.direction === SfScrollableDirection.ho
...
-        v-bind="getNextButtonProps"
-        :disabled="nextDisabled"
-      >
-        <SfIconChevronRight />
-      </SfButton>
     </component>
     <div v-if="$slots.nextButton" v-bind="getNextButtonProps"><slot name="nextButton" /></div>
     <SfButton
-      v-else-if="buttonsPlacement === SfScrollableButtonsPlacement.block"
+      v-else-if="buttonsPlacement"
```
Comment (FRSgit) [inline packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue:123]:
```suggestion
      v-else-if="buttonsPlacement !== SfScrollableButtonsPlacement.none"
```

File: packages/sfui/frameworks/react/components/SfScrollable/SfScrollable.tsx (hunk 1 above)
Comment (AdamPawlinski) [reply packages/sfui/frameworks/react/components/SfScrollable/SfScrollable.tsx]: fixed

File: packages/sfui/frameworks/react/components/SfScrollable/SfScrollable.tsx (hunk 2 above)
Comment (AdamPawlinski) [reply packages/sfui/frameworks/react/components/SfScrollable/SfScrollable.tsx]: fixed

File: packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue (hunk 3 above)
Comment (AdamPawlinski) [reply packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue]: added

File: packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue (hunk 4 above)
Comment (AdamPawlinski) [reply packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue]: changed

File: packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue (hunk 6 above)
Comment (AdamPawlinski) [reply packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue]: added

File: packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue [hunk 7]
```diff
@@ -114,48 +116,19 @@ const isHorizontal = computed(() => props.direction === SfScrollableDirection.ho
...
       variant="secondary"
       size="lg"
       square
       :class="[
         '!rounded-full bg-white hidden md:block',
-        isHorizontal ? 'ml-4' : 'mt-4 rotate-90',
+        isHorizontal && SfScrollableButtonsPlacement.block ? 'ml-4' : 'mt-4 rotate-90',
+        isHorizontal && SfScrollableButtonsPlacement.floating ? 'right-4' : 'bottom-4 rotate-90',
```
[the remark this answers is not in the document]
Comment (AdamPawlinski) [reply packages/sfui/frameworks/vue/components/SfScrollable/SfScrollable.vue]: changed

**Review by FRSgit — APPROVED** [review]

**Comment by Szymon-dziewonski** [comment]
@AdamPawlinski thanks for picking it up :)

**Pull request merged**